layers/finders.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, apps=None, *args, **kwargs):
         layers = list(get_current_layer_stack(get_current_request()))
         layers.reverse()
         processed = []
-        for k, v in self.storages.items():
+        for k, v in list(self.storages.items()):
```

Comment:
I'm not entirely sure how it works, but it essentially creates a copy of `self.storage.items()` so that the `OrderedDict` being iterated isn't the one being modified, eliminating the `"OrderedDict mutated during iteration"` error.